docs/faq/configuration.md

```diff
@@ -59,6 +59,25 @@ On the other hand, increasing your maximum connections, which really just increa
 
 ---
 
+#### 6. How do I run dcrd and dcrwallet as a background daemon?
+
+`dcrd` and `dcrwallet` do not provide a `--daemon` or `--detach` flag to run as a
+daemon as this is considered bad practice. 
+[This blog post](https://www.mikeperham.com/2014/09/22/dont-daemonize-your-daemons/)
+by Mike Perham provides some background information.
+Modern operating systems provide specialized facilities for running processes as
```

Comment:
This might make more sense to someone more familiar with daemons, but this sentence is a bit confusing to me. Suggest breaking it up into smaller, more readable sentences. Below I've taken a naive stab. This is likely not technically correct, but may provide useful scaffolding. 

>Modern operating systems provide specialized facilities for running processes as background daemons. For example, systemd, System V and Upstart are Unix utilities that specialize in running background daemons. This is the recommended method for running `dcrd` or `dcrwallet` as daemons. MacOS X provides a daemon service named [launchd](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPSystemStartup/Chapters/CreatingLaunchdJobs.html).

Reply:
Almost correct, System V is Unix, systemd and Upstart are Linux :wink: 

I'm actually going to remove the System V reference, it's a bit arcane for our target audience here, and most Unix users are probably already familiar with how to run a long-lived daemon.